### PR TITLE
Enable priority sampling by default

### PR DESCRIPTION
**Pull request by brettlangdon**
We want to enable priority sampling by default now.

### CODE REVIEW

File: docs/advanced_usage.rst [hunk 1]
```diff
@@ -128,12 +128,12 @@ priority to the following values:
 - ``AUTO_REJECT``: the sampler automatically rejects the trace
 - ``AUTO_KEEP``: the sampler automatically keeps the trace
 
-Priority sampling is disabled by default. Enabling it ensures that your sampled
-distributed traces will be complete. To enable priority sampling::
+Priority sampling is enabled by default. Enabling it ensures that your sampled
+distributed traces will be complete. To disable priority sampling::
 
-    tracer.configure(priority_sampling=True)
+    tracer.configure(priority_sampling=False)
```
Comment (palazzem) [inline docs/advanced_usage.rst]: Let's remove this snippet. Disabling priority sampling, means breaking distributed tracing so I'd say just to tell people how to use it.

File: ddtrace/tracer.py [hunk 2]
```diff
@@ -110,8 +110,12 @@ def configure(self, enabled=None, hostname=None, port=None, sampler=None,
         if sampler is not None:
             self.sampler = sampler
 
-        if priority_sampling:
+        # If priority sampling is not set or is True and no priority sampler is set yet
+        if priority_sampling in (None, True) and not self.priority_sampler:
```
[the remark this answers is not in the document]
Comment (palazzem) [reply ddtrace/tracer.py]: All good.

File: docs/advanced_usage.rst [hunk 3]
```diff
@@ -128,13 +128,10 @@ priority to the following values:
...
-Priority sampling is disabled by default. Enabling it ensures that your sampled
-distributed traces will be complete. To enable priority sampling::
-
-    tracer.configure(priority_sampling=True)
-
-Once enabled, the sampler will automatically assign a priority to your traces,
+Priority sampling is enabled by default.
+When enabled, the sampler will automatically assign a priority to your traces,
 depending on their service and volume.
+This ensures that your sampled distributed traces will be complete.
```
Comment (brettlangdon) [inline docs/advanced_usage.rst]: @palazzem better?

Comment (palazzem) [reply docs/advanced_usage.rst]: Perfect, thank you!

**Pull request merged**